src/iqm/benchmarks/benchmark_definition.py

```diff
@@ -34,6 +34,52 @@
 from iqm.qiskit_iqm.iqm_provider import IQMBackend, IQMFacadeBackend
 
 
+@dataclass
+class ObservationIdentifier:
+    """Identifier for observations for ease of use
+
+    Attributes:
+        qubit_indices: list containing the indices of the qubits the observation was executed on.
+    """
+
+    qubit_indices: list[int]
+
+    @property
+    def string_identifier(self) -> str:
+        """String version of the qubit indices for ease of use
+
+        Returns:
+            A string of the qubit indices
+        """
+        return str(self.qubit_indices)
+
+
+@dataclass
+class Observation:
```

Comment:
I'm wondering if `Observation` as a class name is going to be a constant source of confusion and require import-aliasing every time IQM Benchmarks is used in the full-stack of IQM software with pulse-level access and experiment automation libraries...

Reply:
There isn't an interface in the open source stack at the moment. I was thinking of prefixing everything with `Benchmark`, basically. `BenchmarkObservation`, `BenchmarkAnalysisResult`, `BenchmarkRunResult`, etc. Simply because this is what people would most likely alias them as when using in the same environment with e.g. the IQM Experiment automation library. I'm not sure if it's the best idea, and yeah it looks long and bulky...

Reply:
This is now changed